### PR TITLE
Bugfix/#1305 Stop multiple sounds

### LINKED ISSUE #1305: Multiple ping sounds in a row
Not a big issue, but it's pretty irritating when you're doing something else on the PC and someone sends multiple PM's and it just keeps spamming that annoying notification sound. One's enough, I know you've sent me messages, don't need a reminder every few seconds. also i heard this was said already, but flashing icon when you get pm'd like with faf would be nice (like it is on python), instead of the small hard-to-notice "2" on faf icon.

### PR DESCRIPTION
**Pull request by Sheikah45**
Fixes #1305

### CODE REVIEW

**Comment by Brutus5000** [comment]
How can delaying without a queue solve this? Don't you just move the problem to a later point?
I would expect we'd need an aggregating function only firing every few seconds.

**Comment by Sheikah45** [comment]
The title might be misleading. It doesnt actually delay. It just wont play any new sounds for 30 seconds after a sound is played

File: src/main/java/com/faforever/client/audio/AudioService.java [hunk 1]
```diff
@@ -21,10 +23,12 @@
   private static final String INFO_SOUND = "theme/sounds/info.mp3";
   private static final String MENTION_SOUND = "theme/sounds/mention.mp3";
   private static final String PRIVATE_MESSAGE_SOUND = "theme/sounds/pm.mp3";
+  private static final long SOUNDS_DELAY = 30000;
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/audio/AudioService.java]: Maybe SILENCE_PERIOD_AFTER_SOUND ?

Comment (Sheikah45) [reply src/main/java/com/faforever/client/audio/AudioService.java]: yeah probably better

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**